server/registration_manager.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 XLAB d.o.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package server
+
+import (
+	"github.com/go-redis/redis"
+)
+
+type registrationManager struct {
+	*redis.Client
+}
+
+func newRegistrationManager(address string) (*registrationManager, error) {
```

Comment:
A solution needs to be found so that Travis tests will pass.

Reply:
The tests failed because there was no registration server available to serve registration keys, and no redis DB to check the registration keys.
This is now half-mocked: Redis DB gets deployed in the travis test, and `getRegistrationKey` method in `pseudonymsys_test` itself inserts the key in the DB instead of querying the registration server.